Mathlib/Geometry/Manifold/ContMDiff/Basic.lean

```diff
@@ -259,6 +259,55 @@ alias contMDiffAt_of_not_mem := contMDiffAt_of_notMem
 alias contMDiffAt_of_not_mem_mulTSupport := contMDiffAt_of_notMem_mulTSupport
 
 
+/-! ### Being `C^k` on a set can be tested in its union -/
```

Comment:
What's "its union"?

Reply:
Good find; I realised the same mid-way through creating all these PRs and forgot to update this one. Is the following phrasing better?
```suggestion
/-! ### Being `C^k` on a union of open sets can be tested on each set -/